techniques/systemSettings/misc/setVars/1.0/metadata.xml

```diff
@@ -0,0 +1,104 @@
+<TECHNIQUE name="Variables (any)">
+
+  <DESCRIPTION>This technique loads rudder variables of type string or dict. They can then be used with a given specific prefix. 
+    Usage: in directives use ${variable_prefix.variable_name}, in mustache {{{vars.variable_prefix.variable_name}}}
+    An unrecognize variable may come from a misordering of your rules executions, the execution follows an alphabetical ordre, based on rules names.
+  </DESCRIPTION>
+  <MULTIINSTANCE>true</MULTIINSTANCE>
+
+
+  <TMLS>
+    <TML name="setVars"/>
+  </TMLS>
+
+  <BUNDLES>
+    <NAME>setVars</NAME>
+  </BUNDLES>
+
+  <TRACKINGVARIABLE>
+    <SAMESIZEAS>SET_VARS_NAME</SAMESIZEAS>
+  </TRACKINGVARIABLE>
+
+   <SYSTEMVARS>
+     <NAME>SHARED_FILES_FOLDER</NAME>
+   </SYSTEMVARS>
+
+
+  <SECTIONS>
+    <SECTION name="Load the variables" multivalued="true" component="false">
+      <SECTION name="Load rudder variables" multivalued="false" component="true" componentKey="SET_VARS_NAME">
```

Comment:
Reporting is way easier this way for multivalued/multiInstance case. Because you only have one trackingKey.